api/src/utils/sanitize-query.ts

```diff
@@ -98,7 +98,12 @@ function sanitizeFilter(rawFilter: any, accountability: Accountability | null) {
 
 	filters = deepMap(filters, (val) => {
 		try {
-			return JSON.parse(val);
+			const parsedObject = JSON.parse(val);
+			if (parsedObject == 'number' && typeof val == 'string' && val.length > 15) {
```

Comment:
You are comparing the returned value with string `number`.

Why not to use `Number.isSafeInteger` ?

Reply:
Oh dear, it's supposed to be `typeof parsedObject`. ><

Reply:
Don't you want to try with `Number.isSafeInteger`? It is more clean and explicit.

Reply:
Wow, I didn't know that `Number.isSafeInteger` exists. 👍🏻 Pushing a new fix now~

Reply:
Pushed. Thank you @joselcvarela~

Reply:
> Wow, I didn't know that Number.isSafeInteger exists. 👍🏻 Pushing a new fix now~

TIL! Didn't know that was a thing either 🙂